packages/web-frontend/src/sagas.js

```diff
@@ -204,9 +205,8 @@ function* handleUserPage(userPage, initialComponents) {
 const URL_REFRESH_COMPONENTS = {
   [URL_COMPONENTS.PROJECT]: setProject,
   [URL_COMPONENTS.ORG_UNIT]: setOrgUnit,
-  [URL_COMPONENTS.MAP_OVERLAY]: setMapOverlays,
+  [URL_COMPONENTS.MAP_OVERLAY]: setMapOverlaysOnceHierarchyLoads,
   [URL_COMPONENTS.REPORT]: openEnlargedDialog,
-  [URL_COMPONENTS.OVERLAY_PERIOD]: updateCurrentOverlayConfigOnceHierarchyLoads,
```

Comment:
Previously we fetch twice request when we refresh page, here is why because we loop every entry in URL_REFRESH_COMPONENTS to set state, which will call requests. 
I've removed `OVERLAY_PERIOD` and keep `MAP_OVERLAY`. Now `setMapOverlaysOnceHierarchyLoads` will set overlay and overlay period before senting a request for measureData.